app/api/dao/user.py

```diff
@@ -165,6 +167,16 @@ def list_users(user_id, search_query="", is_verified=None):
             )
         ]
 
+        filter_by_name = lambda list_of_users: list_of_users['name']
+
+        if sort_order!='':
+            if sort_order=='1':
+                list_of_users.sort(key=filter_by_name)
```

Comment:
Why are we filtering it here when we can just filter it while querying the data from sql?